coordinator/internal/logic/submitproof/proof_receiver.go

```diff
@@ -332,21 +336,28 @@ func (m *ProofReceiverLogic) updateProofStatus(ctx context.Context, proverTask *
 				log.Error("failed to update batch proving_status as failed", "hash", proverTask.TaskID, "error", err)
 				return err
 			}
+		case message.ProofTypeBundle:
+			if err := m.bundleOrm.DecreaseActiveAttemptsByHash(ctx, proverTask.TaskID, tx); err != nil {
+				log.Error("failed to update bundle proving_status as failed", "hash", proverTask.TaskID, "error", err)
+				return err
+			}
 		}
 
 		// if the block batch has proof verified, so the failed status not update block batch proving status
-		if m.checkIsTaskSuccess(ctx, proverTask.TaskID, proofMsg.Type) {
+		if m.checkIsTaskSuccess(ctx, proverTask.TaskID, message.ProofType(proofParameter.TaskType)) {
 			log.Info("update proof status skip because this chunk/batch has been verified", "hash", proverTask.TaskID, "public key", proverTask.ProverPublicKey)
 			return nil
 		}
 
 		if status == types.ProverProofValid {
 			var storeProofErr error
-			switch proofMsg.Type {
+			switch message.ProofType(proofParameter.TaskType) {
 			case message.ProofTypeChunk:
-				storeProofErr = m.chunkOrm.UpdateProofAndProvingStatusByHash(ctx, proofMsg.ID, proofMsg.ChunkProof, types.ProvingTaskVerified, proofTimeSec, tx)
+				storeProofErr = m.chunkOrm.UpdateProofAndProvingStatusByHash(ctx, proofParameter.TaskID, []byte(proofParameter.Proof), types.ProvingTaskVerified, proofTimeSec, tx)
```

Comment:
this will break the proof field. previously, it's:
```
// ChunkProof includes the proof info that are required for chunk verification and rollup.
type ChunkProof struct {
	StorageTrace []byte `json:"storage_trace,omitempty"`
	Protocol     []byte `json:"protocol"`
	Proof        []byte `json:"proof"`
	Instances    []byte `json:"instances"`
	Vk           []byte `json:"vk"`
	// cross-reference between cooridinator computation and prover compution
	ChunkInfo  *ChunkInfo           `json:"chunk_info,omitempty"`
	GitVersion string               `json:"git_version,omitempty"`
	RowUsages  []SubCircuitRowUsage `json:"row_usages,omitempty"`
}
```
after change, it's only `Proof        []byte `json:"proof"`` in `ChunkProof` type.

Reply:
Currently change is the same as before. coordinator just trust the prover proof bytes and insert to db.